TTS/tts/models/overflow.py

```diff
@@ -74,6 +77,9 @@ def __init__(
         for key in config:
             setattr(self, key, config[key])
 
+        if not hasattr(self, "neural_hmm_tts_only"):
+            self.neural_hmm_tts_only = False
```

Comment:
zoo tests are still failing, how do we fix those?

Reply:
You can check the logs at the bottom of the failed run. 
I think your updates in the overflow model make it incompatible with the released model.

Reply:
It might be better to implement this model separately to prevent such conflict for now and the future.

Reply:
Here is the link to the logs if helps

https://github.com/coqui-ai/TTS/actions/runs/3873294513/jobs/6603145674

Reply:
Yeah! I think I will implement it as a separate model.